lib/setup.js

```diff
@@ -1516,7 +1527,7 @@ function processCommand(command, args, params, callback) {
             tools.getRepositoryFile(null, null, (_err, sources, _sourcesHash) => {
                 if (sources) {
                     for (const s in sources) {
-                        if (sources.hasOwnProperty(s)) {
+                        if (Object.prototype.hasOwnProperty.call(sources, s)) {
```

Comment:
For future updates we could start changing these constructs to one of
* `for (const [s, source] of Object.entries(sources))`
* `for (const s of Object.keys(sources))`
* `for (const source of Object.values(sources))`

Reply:
:+1: